index.html

```diff
@@ -2829,10 +2829,66 @@
           <li>
             <a>Queue a task</a> on the <a>user interaction task source</a> to
             run the following steps:
-            <ol>
+            <ol data-link-for="PaymentAddress">
+              <li>Let <var>address</var> be a new instance of
+              <a>PaymentAddress</a>.
+              </li>
+              <li>Set the <a>addressLine</a> attribute of <var>address</var> to
+              the result of splitting the end user's address line into a
+              <a data-cite="!WEBIDL#dfn-frozen-array-type">frozen array</a>. If
+              none was provided, set it to the empty <a data-cite=
+              "!WEBIDL#dfn-frozen-array-type">frozen array</a>.
+                <aside class="note">
+                  How to split an address line is locale dependent and beyond
+                  the scope of this specification.
+                </aside>
+              </li>
+              <li>Set the <a>country</a> attribute of <var>address</var> to the
+              end user's country as an upper case [[!ISO3166]] alpha-2 code or
+              to the empty string if none was provided.
+              </li>
+              <li>If <var>request</var>.<a>[[\options]]</a>.<a data-lt=
+              "PaymentOptions.requestPayerPhone">requestPayerPhone</a> is
+              false, set the <a>phone</a> attribute of <var>address</var> to
+              the empty string. Otherwise, set the <a>phone</a> attribute of
+              <var>address</var> to a user provided phone number, optionally
+              formatted to adhere to [[!E.164]], or to the empty string if none
+              was provided.
+              </li>
+              <li>Set the <a>languageCode</a> attribute of <var>address</var>
+              to a <a data-cite="!BCP47#section-4.5">canonicalized language
+              tag</a>, or to the empty sting if none was provided.
```

Comment:
"sting" -> "string"

Do we really expect a language to be "provided"? We might want to expand on this somehow.

Reply:
So, this opened this interesting can of worms: https://github.com/w3c/payment-request/issues/608 ... it might be simple to solve, but I need to look in detail at how Chrome implements this via the Unicode CLDR library. 

This area is new to me though, so will need to do a bit of research.

Reply:
Yeah... maybe mark it with a warning class or something linking to that issue in the meantime?